### PR TITLE
Updated README for new testnet

### PR DESCRIPTION
**Pull request by fedekunze**
Note: Merge only once the new testnet has been released. You'll probably have to change L81 and L83 with the new name for the testnet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,74 +38,59 @@ sudo apt-get install wget
...
 
-You can find the corresponding genesis files [here](https://github.com/tendermint/testnets). Then replace the `genesis.json`and `config.toml` files:
+You can find the corresponding genesis files [here](https://github.com/cosmos/testnets). Then replace the `genesis.json` and `config.toml` files:
 
 ```
-rm $HOME/.gaiad/config/genesis.json $HOME/.gaiad/config/config.toml $HOME/.gaiad/config/addrbook.json
-
 wget -O $HOME/.gaiad/config/genesis.json https://raw.githubusercontent.com/cosmos/testnet/master/gaia-4000/genesis.json
 
 wget -O $HOME/.gaiad/config/config.toml https://raw.githubusercontent.com/cosmos/testnet/master/gaia-4000/config.toml
```
Comment (rigelrozanski) [inline README.md]: Are we going to update this to the new testnet? I guess as soon as we release it - we should update simultaneously

File: README.md [hunk 2]
```diff
@@ -219,14 +227,20 @@ gaiacli validatorset
 
 **Note:** Remember that to be in the validator set you need to have more total power than the Xnd validator, where X is the assigned size for the validator set \(by default _`X = 100`_\).
 
-#### Delegating: Bonding and unbonding to a validator
+## Delegate your tokens
```
[the remark this answers is not in the document]
Comment (rigelrozanski) [reply README.md]: ?

File: README.md [hunk 3]
```diff
@@ -235,19 +249,9 @@ gaiacli unbond --address-delegator=$MYADDR --address-candidate=<bonded_validator
 
 You can unbond a specific amount of`shares`\(eg:`12.1`\) or all of them \(`MAX`\).
 
-You should now see the unbonded tokens reflected in your balance and in your delegator bond :
+You should now see the unbonded tokens reflected in your balance and in your delegator bond:
 
 ```
 gaiacli account $MYADDR
```
Comment (rigelrozanski) [inline README.md]: Need to specify `--chain-id`

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gamarin2) [reply README.md]: Ex: Use "Delegating, Bonding, Becoming" or "Delegate, Bond, Become" in section titles

File: README.md (hunk 1 above)
Comment (fedekunze) [reply README.md]: Idk yet which will be the name for the new testnet...

File: README.md [hunk 4]
```diff
@@ -1,100 +1,132 @@
...
+# Connect to a Testnet
 
-## Install Cosmos-SDK on a Digital Ocean Droplet \(Recommended\)
+This document explains how to connect to the Testnet of a [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/) based blockchain. It can be used to connect to the latest Testnet for the Cosmos Hub.
+
+## Software Setup (Manual Installation)
+
+Follow this instructions to install the Cosmos-SDK and connect to the latest Testnet. This instructions work for both a local machine and a VM in a cloud server.
+
+If you intend to run a non-validator full-node, installing the SDK on a Cloud server is a good option. Although, if you are want to become a validator for the `mainnet` you should look at more complex setups to protect your node from DDOS and ensure high-availability (see the [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements)). You can check more information regarding validators in our [website](https://cosmos.network/validators) or in the [validator FAQ](https://cosmos.network/resources/validator-faq).
```
Comment (gamarin2) [inline README.md]: Although -> However

Comment (gamarin2) [inline README.md]: You can get more information regarding validators in our website, in the Validator FAQ or in the Validator Chat
(add chat)

File: README.md [hunk 5]
```diff
@@ -160,46 +197,26 @@ Now check the destination account and your own account to check the updated bala
...
-
-You can also define a custom fee on the transaction by adding the `--fee` flag using the same format:
-
-```
-gaiacli send --from=$MYADDR --amount=1000fermion --fee=1fermion --chain-id=<name_of_testnet_chain> --sequence=1 --name=$KEYNAME --to=<destination_address>
-```
+## Run a Validator Node
 
-### Transfer tokens to other chain
+[Validators](https://cosmos.network/validators) are actors from the network that are responsible from commiting new blocks to the blockchain by submitting their votes. This section covers the instructions necessary to stake tokens to become a testnet validator candidate.
```
Comment (gamarin2) [inline README.md]: Add a little reminder about sentry node architecture

File: README.md (hunk 1 above)
Comment (rigelrozanski) [reply README.md]: I think we're settting up gaia5000 today

File: README.md [hunk 6]
```diff
@@ -1,8 +1,64 @@
-# Deploy a Testnet
+# Connect to a Testnet
 
-## Software Setup \(Manual Installation\)
+This document explains how to connect to the Testnet of a [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/) based blockchain. It can be used to connect to the latest Testnet for the Cosmos Hub.
 
-- Install [GNU Wget](https://www.gnu.org/software/wget/):
+## Software Setup (Manual Installation)
+
+Follow this instructions to install the Cosmos-SDK and connect to the latest Testnet. This instructions work for both a local machine and a VM in a cloud server.
```
Comment (gamarin2) [inline README.md]: these

Comment (gamarin2) [reply README.md]: these

File: README.md [hunk 7]
```diff
@@ -1,8 +1,64 @@
...
+
+If you intend to run a non-validator full-node, installing the SDK on a Cloud server is a good option. However, if you are want to become a validator for the `mainnet` you should look at more complex setups, like a [Sentry Node Architecture](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#how-can-validators-protect-themselves-from-denial-of-service-attacks), to protect your node from DDOS and ensure high-availability (see the [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements)). You can check more information regarding validators in our [website](https://cosmos.network/validators), in the [Validator FAQ](https://cosmos.network/resources/validator-faq) or in the [Validator Chat](https://riot.im/app/#/room/#cosmos_validators:matrix.org).
```
Comment (gamarin2) [inline README.md]: if you want

Comment (gamarin2) [reply README.md]: Hub's `mainnet`

Comment (gamarin2) [reply README.md]: like a -> including sentry node architectures

Comment (gamarin2) [reply README.md]: You can find more information on validators

Comment (gamarin2) [reply README.md]: or in -> and in

File: README.md [hunk 8]
```diff
@@ -1,8 +1,64 @@
...
+go get -u all
+```
+
+If you want to update a specific package run:
+
+```
+go get -u <package_name>
+```
+
+### Glide
```
Comment (gamarin2) [inline README.md]: Install Glide + link

Comment (zmanian) [reply README.md]: What software still requires Glide?

Comment (fedekunze) [reply README.md]: I thought it was still using Glide

File: README.md [hunk 9]
```diff
@@ -70,11 +130,8 @@ gaiad init
 You can find the corresponding genesis files [here](https://github.com/tendermint/testnets). Then replace the `genesis.json`and `config.toml` files:
```
Comment (gamarin2) [inline README.md]: This deserves an explanation. 
- Explain what the genesis files are for, and how to choose the correct ones. "Genesis files are the starting point for the full-node to sync up with the network. In order to sync up with the correct version of the Testnet, be sure to choose the genesis file corresponding to the version of the Testnet you want to connect to."
- Short explanation of what the config.toml file is for.

File: README.md [hunk 10]
```diff
@@ -138,22 +215,67 @@ gaiacli account <destination_address>
...
+You can also check your balance at a given block by using the `--block` flag:
+
+```
+gaiacli account <your_address> --block=<block_height>
+```
+
+## Run a Validator Node
 
-## Staking: Becoming a Validator
+[Validators](https://cosmos.network/validators) are actors from the network that are responsible from commiting new blocks to the blockchain by submitting their votes. This section covers the instructions necessary to stake tokens to become a testnet validator candidate.
```
Comment (gamarin2) [inline README.md]: Missing little reminder for people that intend to run validators in the future mainnet that sentry node architecture is strongly recommended

File: README.md (hunk 8 above)
Comment (zmanian) [reply README.md]: The SDK only requires `dep`

**Comment by rigelrozanski** [comment]
@fedekunze @zmanian not sure if we want to pull the `genesis.json` directly or create it was the commands I've outlined here https://gist.github.com/rigelrozanski/306364f86e9a2918ebde19df9fdb78dc

File: README.md [hunk 11]
```diff
@@ -56,36 +98,33 @@ gaiacli version
...
 
-You can find the corresponding genesis files [here](https://github.com/tendermint/testnets). Then replace the `genesis.json`and `config.toml` files:
+Get the genesis transactions
 
 ```
-rm $HOME/.gaiad/config/genesis.json $HOME/.gaiad/config/config.toml $HOME/.gaiad/config/addrbook.json
+git clone github.com/cosmos/testnets
 
-wget -O $HOME/.gaiad/config/genesis.json https://raw.githubusercontent.com/cosmos/testnet/master/gaia-4000/genesis.json
+cp -a testnets/gaia-5000/. $HOME/.gaiad/config/gentx
```
Comment (rigelrozanski) [inline README.md]: This should use absolute directories aka: $GOPATH/src/github.com/cosmos/testnets/gaia-5000/.

**Comment by nylira** [comment]
gaia-5000 is already running on 10 nodes. I am +1 for merging this readme so it goes on the website and then improving it as we go.

File: README.md (hunk 11 above)
Comment (zmanian) [reply README.md]: Do you want to use `git clone` or `go get` as a way of cloning the repo?

I think `git clone` will be better for devs who don't GOLANG 💯

**Comment by zmanian** [comment]
@rigelrozanski Can you contribute updates to the staking section?

File: README.md (hunk 11 above)
Comment (rigelrozanski) [reply README.md]: yeah git clone makes more sense - I just use go get cause I'm laZ

Comment (rigelrozanski) [reply README.md]: should still add the instruction and ideally use absolute paths to the clone directory

**Comment by ebuchman** [comment]
I'm going to udate and merge this ...

**Pull request merged**